### DISCUSSION TITLE
Rename Livewire internal properties `$data` and `$errors`

### DISCUSSION THREAD

**Discussion by adevade** [livewire/livewire#4837] [3. Feature ideas]
Sometimes they're just the best name for properties on your component, but if you try to use them now Livewire (can) break down. (Right?)

I suggest that for the next major version, these internal properties be renamed to something else. Doesn't matter what really, since the user should not mess with them anyway?

Shouldn't even need to be a breaking change, no? Since no one should modify these properties directly?

Some suggestions: 
- `$_data` & `$_errors`
- `$_livewireData` & `$_livewireErrors`
- `$_wireData` & `$_wireData`
- `$_lwData` & `$_lwErrors`

What do you think?

**Comment by joshhanley** [comment]
@adevade what is the `$data` property you are talking about?

With `$errors` it is actually the same as Laravel, so it I think it's better to keep it consistent.

Hope this helps!
